### DISCUSSION TITLE
Termux and Shizuku?

### DISCUSSION THREAD

**Discussion by GL513** [RikkaApps/Shizuku#224] [Q&A] [answered]
Is there a way to use Termux to locally interact with shizuku and run commands from termux and through to shizuku, like ADB? Obviously, wireless ADB is already a thing, but it would be cool to not have to be connected to WiFi, plus, you would be able to run some superuser-like commands, correct?

**Comment by haruue** [comment, marked as answer]
Isn't this what you need?

![out](https://user-images.githubusercontent.com/13131972/176204578-c530706d-69d4-42ad-b5a3-c9bb82d9d7c6.jpg)

**Reply by GL513** [reply]
Alright yes, thank you 😅
I forgot that was a feature.
